addons/main/script_macros.hpp

```diff
@@ -183,13 +186,14 @@
 #undef GET_SM_STATE
 #define GET_SM_STATE(_unit)         ([_unit, ACEGVAR(medical,STATE_MACHINE)] call CBA_statemachine_fnc_getCurrentState)
 
+#undef GET_BLOOD_VOLUME             
+
 #define GET_OPIOID_FACTOR(unit)           (unit getVariable [QEGVAR(pharma,opioidFactor), 1])
+#define GET_PAIN_PERCEIVED(unit)    (0 max ((GET_PAIN(unit) - GET_PAIN_SUPPRESS(unit)) min 1))
```

Comment:
we had it defined below, why changing its place?

Reply:
I did not Ctrl+F and thought it wasn't there, so I rewrote it before I saw it and just shifted it